lib/model/content.dart

```diff
@@ -872,6 +870,53 @@ class _ZulipContentParser {
     return descendant4.text.trim();
   }
 
+  UserMentionNode? parseUserMention(dom.Element element) {
+    assert(_debugParserContext == _ParserContext.inline);
+    assert(element.localName == 'span');
+    final debugHtmlNode = kDebugMode ? element : null;
+
+    final classes = element.className.split(' ')..sort();
+    assert(classes.contains('user-mention')
+        || classes.contains('user-group-mention'));
+    int i = 0;
+
+    if (i >= classes.length) return null;
+    bool isChannelWildcardClassIncluded = false;
```

Comment:
nit: can simplify "is X included" to "has X":
```suggestion
    bool hasChannelWildcardClass = false;
```